src/frontend/src/pages/PPQPage/PPQPage.scss

```diff
@@ -26,15 +37,14 @@
   align-items: end;
 }
 
-.list {
-  padding-bottom: 15rem;
-  padding-left: 0.9rem;
-}
-
 .ppq-svg > img {
   max-width: 600px;
 }
 
 .col-md-6 img {
   width: 100%;
 }
+
+.intake-heading {
```

Comment:
If the only thing that this class will do is apply  `font-weight: 700`, then lets add it to a higher-level file and name it something more generic that aligns with our CSS Standards. Documentation on these standards may be found in our wiki.